doc/user/layouts/shortcodes/self-managed/terraform-disclaimer.html

```diff
@@ -0,0 +1,9 @@
+The sample Terraform module is for **evaluation purposes only** and not intended
+for production use. It is provided to help you get started with Materialize for
+evaluation purposes only. Materialize does not support nor recommends this
+module for production use. Materialize does not guarantee tests for changes to
+the module.
```

Comment:
not sure what "guarantee tests for changes" is

Reply:
Updated in patch